cinema/views.py

```diff
@@ -48,6 +79,35 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
u have already implemented this method u can write a mixin or make it as standalone func to make ur code dry